.github/workflows/build_and_test.yml

```diff
@@ -26,4 +26,20 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           config: .github/.licenserc.yaml
-
+  test_ci:
+    name: "Test CI"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java-version: [ 11, 17, 21 ]
```

Comment:
Please remove Java 11 and make a spin-off PR for this `build_and_test.yml`, @jiangzho .